metafix/src/main/java/org/metafacture/metafix/FixMethod.java

```diff
@@ -680,7 +680,13 @@ public void apply(final Metafix metafix, final Record record, final List<String>
     to_base64 {
         @Override
         public void apply(final Metafix metafix, final Record record, final List<String> params, final Map<String, String> options) {
-            record.transform(params.get(0), s -> Base64.getEncoder().encodeToString(s.getBytes()));
+            final boolean urlSafe = getBoolean(options, "url_safe");
+            if (!urlSafe) {
+                record.transform(params.get(0), s -> Base64.getEncoder().encodeToString(s.getBytes()));
+            }
+            else {
+                record.transform(params.get(0), s -> Base64.getUrlEncoder().encodeToString(s.getBytes()));
+            }
```

Comment:
Prefer not to repeat application logic.

```suggestion
            final boolean urlSafe = getBoolean(options, "url_safe");
            final Base64.Encoder encoder = urlSafe ? Base64.getUrlEncoder() : Base64.getEncoder();

            record.transform(params.get(0), s -> encoder.encodeToString(s.getBytes()));
```

Reply:
ACK